src/pages/Contacts.jsx

```diff
@@ -0,0 +1,85 @@
+// React Imports
+import React, { useState } from 'react';
+// Material UI Imports
+import AddIcon from '@mui/icons-material/Add';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Container from '@mui/material/Container';
+import Typography from '@mui/material/Typography';
+// Component Imports
+import { useContactsList } from '@hooks';
+import { AddContactModal, DeleteContactModal } from '@components/Modals';
+import { ContactListTable } from '@components/Contacts';
+import { LoadingAnimation, EmptyListNotification } from '@components/Notification';
+
+/**
+ * Contacts Component - Component that generates Contacts Page for PASS
+ *
+ * @memberof Pages
+ * @name Contacts
+ */
+
+const Contacts = () => {
+  localStorage.setItem('restorePath', '/contacts');
+  // state for AddContactModal component
```

Comment:
Not sure we need the comment for this state, if we aren't going to be consistent and label the other states as well. I think the name of the state is self-documenting enough honestly.

Reply:
fixed